x-pack/plugins/observability_solution/slo/docs/openapi/slo/bundled.json

```diff
@@ -1251,6 +1304,11 @@
                 "type": "string",
                 "example": "my-service-*"
               },
+              "dataViewId": {
+                "description": "The kibana data view id to use, primarily used to include data view runtime mappings. Make sure to save SLO again if you add/update run time fields to the data view and if those fields are being used in slo queries.",
```

Comment:
To be honest I've never used runtime fields in the index selector popup. But this is a built in feature of the DataView component, so this sounds correct. My only concern was about the fact that you mentioned its primary use is to include data view runtime mappings. 

<img width="1504" alt="Screenshot 2024-06-20 at 22 30 36" src="https://github.com/elastic/kibana/assets/2852703/253745cd-7184-42ab-91ca-76c36523ddf3">

Reply:
This text was actually changed in https://github.com/elastic/kibana/pull/176662. It can certainly be updated, I'm just not sure of the history or details.